lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -83,6 +84,7 @@ def expected_name(method_name, prefix)
           new_name << '?' unless method_name.end_with?('?')
           new_name
         end
+        # rubocop:enable InternalAffairs/MethodNameEndWith
```

Comment:
It can probably narrow the scope of these disable comments. The positions will be clearer.

```diff
diff --git a/lib/rubocop/cop/naming/predicate_name.rb b/lib/rubocop/cop/naming/predicate_name.rb
index 740166125..fbbad0f26 100644
--- a/lib/rubocop/cop/naming/predicate_name.rb
+++ b/lib/rubocop/cop/naming/predicate_name.rb
@@ -66,12 +66,11 @@ module RuboCop

         private

-        # rubocop:todo InternalAffairs/MethodNameEndWith
         def allowed_method_name?(method_name, prefix)
           !(method_name.start_with?(prefix) && # cheap check to avoid allocating Regexp
               method_name.match?(/^#{prefix}[^0-9]/)) ||
             method_name == expected_name(method_name, prefix) ||
-            method_name.end_with?('=') ||
+            method_name.end_with?('=') || # rubocop:todo InternalAffairs/MethodNameEndWith
             allowed_method?(method_name)
         end

@@ -81,10 +80,10 @@ module RuboCop
                      else
                        method_name.dup
                      end
-          new_name << '?' unless method_name.end_with?('?')
+          new_name << '?' unless method_name.end_with?('?') # rubocop:todo InternalAffairs/MethodNameEndWith
+
           new_name
         end
-        # rubocop:enable InternalAffairs/MethodNameEndWith

         def message(method_name, new_name)
           "Rename `#{method_name}` to `#{new_name}`."
```

Reply:
And `rubocop:disable` may be preferable to `rubocop:todo` if there is no fix.

Reply:
It's nothing that can't be fixed, but these methods are mostly concerned with `method_name`, and it seemed a bit off to change them to receive `node` as argument to conform to this cop.
I couldn't decide whether to make a change or not, so I compromised with todo.